internal/daemon/controller/auth/auth.go

```diff
@@ -945,3 +947,11 @@ func (r *VerifyResults) ScopesAuthorizedForList(ctx context.Context, rootScopeId
 func (r *VerifyResults) GrantsHash(ctx context.Context) ([]byte, error) {
 	return r.grants.GrantHash(ctx)
 }
+
+func GetRequestInfo(ctx context.Context) (*authpb.RequestInfo, bool) {
```

Comment:
Please add a comment above this function to explain how it's used.
```suggestion
// GetRequestInfo extracts the request info stored in the context, if it exists.
// This returns nil, false if the request info could not be found.
func GetRequestInfo(ctx context.Context) (*authpb.RequestInfo, bool) {
```